build-openjdk8.sh

```diff
@@ -56,6 +62,13 @@ build() {
       target="images"
     fi
     make LOG_LEVEL=debug CONF=$debug $target
+    if [ "${debug}_" == "release-jfr_" ]; then
+      TARBALL_NAME="${TARBALL_BASE_NAME}-jdk-jfr_${PLATFORM_VERSION}"
```

Comment:
I'd prefer if we kept global config variables at the top of the script. So the suggestion would be to define `TARBALL_NAME_JFR` and `TARBALL_NAME_JFR_JRE` and use them in a local variable, say `archive_name` in the script like:

```
archive_name="$TARBALL_NAME"
jre_archive_name="$TARBALL_NAME_JRE"
if [ "${debug}_" == "release-jfr_" ]; then
  archive_name="$TARBALL_NAME_JFR"
  jre_archive_name="$TARBALL_NAME_JFR_JRE"
fi
```